frontend/src/pages/Register/index.jsx

```diff
@@ -103,7 +103,7 @@ const Register = () => {
   };
 
   const validPasswordFormat = (_password) => {
-    const re = /^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#\$%\^&\*])(?=.{8,})/; // eslint-disable-line no-useless-escape
+    const re = /^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#-_\$%\^&\*\?\€\,\;\.\:])(?=.{8,})/; // eslint-disable-line no-useless-escape
```

Comment:
Same as above: `-` should be escaped
```suggestion
    const re = /^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#\-_\$%\^&\*\?\€\,\;\.\:])(?=.{8,})/; // eslint-disable-line no-useless-escape
```